lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -511,6 +511,10 @@ def configure(conf)
             ' Stackdriver output plugin due to a transient error and were'\
             ' retried')
         @ok_code = @use_grpc ? GRPC::Core::StatusCodes::OK : 200
+        unless @successful_requests_count
```

Comment:
You should be able to ask for the registry `name` and complain if it's `nil`... Otherwise you'll get more errors when trying to update the counters (e.g., `NoMethodError`).